pkg/helmclient/pull_chart_tarball.go

```diff
@@ -126,6 +126,7 @@ func (c *Client) newRequest(method, url string) (*http.Request, error) {
 	}
 
 	req.Header.Set("Accept", "application/json")
+	req.Header.Set("Cache-Control", "no-cache")
```

Comment:
Thanks Yasser, this is a nice solution and simpler than the one proposed in the issue. Our charts are served via GitHub Pages so they should respect this header.

One line fixes are the best fixes :)